src/Api.hs

```diff
@@ -34,6 +35,11 @@ data DatumApi route = DatumApi
             -- TODO: change to JSON request
             :> ReqBody '[JSON] GetDatumsByHashesRequest
             :> Get '[JSON] GetDatumsByHashesResponse
+    , getLastBlock ::
+        route
+            :- "block"
+            :> Summary "Get currenctly processed block"
```

Comment:
There's a typo: "currenctly"

Reply:
That summary is actually wrong, `Get latest processed block` is correct one, datums from returned block are guaranteed to be already in database